app/views/checkout/payment.scala.html

```diff
@@ -1,16 +1,22 @@
-@(form: Form[model.SubscriptionData])
+@(form: Form[model.SubscriptionData], userIsSignedIn: Boolean)
 
-@import configuration.Config
+@import configuration.Config.Identity._
+@import routes.Checkout.renderCheckout
 
 @main("Checkout | Subscriptions | The Guardian") {
 
+
 <div class="gs-container">
     @fragments.page.header("Checkout")
 
-
-    <div class="page-section__content">
+    <div class="page-section__content" data-user-signedin="@userIsSignedIn">
+    @if(userIsSignedIn) {
+        <h3 class="h-section h-section--lead">Not you?</h3>
```

Comment:
You don't have any of these classes in your styles, e.g., `h-section-*`, `.page-section__content`. Can you streamline this markup at all?. `page-section__content` should definitely go as you don't appear to have a parent `page-section` anywhere.